src/utils/token-metadata.utils.ts

```diff
@@ -32,6 +33,43 @@ export const loadTokenMetadata$ = memoize(
   { cacheKey: ([address, id]) => getTokenSlug({ address, id }) }
 );
 
+type DetailedAssetMetdata = {
+  decimals: number;
+  symbol: string;
+  name: string;
+  shouldPreferSymbol?: boolean;
+  thumbnailUri?: string;
+  displayUri?: string;
+  artifactUri?: string;
+};
```

Comment:
Take a look at `TokenMetadataResponse`.  
`tokenMetadataApi.get<TokenMetadataResponse>(/metadata/${address}/${id} `
returns the same type that 
`tokenMetadataApi.post<Array<DetailedAssetMetdata | null>>('/', slugs) does`

Fix `TokenMetadataResponse` interface and reuse it